test/utils/test_convert.py

```diff
@@ -611,3 +612,231 @@ def test_from_dgl_hetero_graph():
     assert data['v2'].num_nodes == g.num_nodes('v2')
     assert torch.equal(data['v1'].x, g.nodes['v1'].data['x'])
     assert torch.equal(data['v2'].x, g.nodes['v2'].data['x'])
+
+
+@withPackage('networkx')
+def test_from_hetero_networkx():
+    edge_index = get_random_edge_index(5, 10, 20, coalesce=True)
+    graph_x = [torch.tensor([0, 1, 2])]
+
+    data = HeteroData()
+    data['global_id'] = 0
+    data['graph_x'] = graph_x
+    data['author'].x = torch.arange(5)
+    data['paper'].x = torch.arange(10)
+    data['author', 'paper'].edge_index = edge_index
```

Comment:
Add another edge type , to make sure the edge_index gets offset correctly.

Reply:
Alright, I will be adding three more cases:
- Multi-edge between the same type of nodes, with a different edge type
- Opposite direction edge type (from paper to author for example)
- New edge type between author and another node type (e.g. institution)